dclab/definitions/meta_const.py

```diff
@@ -26,10 +25,12 @@
         ["crosstalk fl32", float, "Fluorescence crosstalk, channel 3 to 2"],
         ["crosstalk fl13", float, "Fluorescence crosstalk, channel 1 to 3"],
         ["crosstalk fl23", float, "Fluorescence crosstalk, channel 2 to 3"],
+        # qpi-related calculations
+        ["qpi alpha RBC", float, "Specific refractive increment [ml/g]"],
+        ["qpi alpha non-RBC", float, "Specific refractive increment [ml/g]"],
```

Comment:
I added two alpha values here (for now). Could be that we need one for each general cell type (e.g. HL60).

Reply:
I think this is too specific to be handled by the dclab configuration, because it also requires third-party software to identify which event is which cell type. The `CFG_ANALYSIS` is only designed for things that dclab can do by itself (e.g. in an ancillary feature). If we are talking about `ml_score` features, then I would add this to `meta_logic`, replace `alpha` with `refinc` (more accurate) and use three-character lower-case identifiers.

Maybe this is something that we should talk about in a later PR...

Reply:
https://github.com/DC-analysis/dclab/issues/192